go/libkb/resolve.go

```diff
@@ -171,6 +171,7 @@ func (r *Resolver) resolveURLViaServerLookup(au AssertionURL, input string, with
 
 	var them *jsonw.Wrapper
 	if them, res.err = ares.Body.AtKey("them").ToArray(); res.err != nil {
+		res.err = NotFoundError{fmt.Sprintf("User not found for query: %q", input)}
```

Comment:
I don't think I understand the context here.  Is there a case where the API server is returning data for a not found user?  Doesn't line 166 handle this already?